applications/tari_base_node/src/bootstrap/base_node.rs

```diff
@@ -157,6 +157,7 @@ where B: BlockchainBackend + 'static
                     pruning_horizon: config.pruning_horizon,
                     orphan_db_clean_out_threshold: config.orphan_db_clean_out_threshold,
                     max_randomx_vms: config.max_randomx_vms,
+                    blocks_behind_before_considered_lagging: self.config.blocks_behind_before_considered_lagging,
```

Comment:
Suggest add info log message after the `StackBuilder` instruction:
```
info!(target: LOG_TARGET, "Blocks behind before considered lagging se to {}", self.config.blocks_behind_before_considered_lagging);
```

Reply:
It logs info, every-time it receives a metadata and its behind.